src/viser/client/src/WorldTransformUtils.ts

```diff
@@ -15,3 +15,26 @@ export function computeT_threeworld_world(viewer: ViewerContextContents) {
     )
     .setPosition(position[0], position[1], position[2]);
 }
+
+/** Helper for converting a ray from the three.js world frame to the Python
+ * world frame. Applies the transformation from computeT_threeworld_world.
+ */
+export function ray_to_viser_coords(
```

Comment:
nit: it would be nice to switch to the camelCase convention we're using for Typescript!

Reply:
Addressed!